src/Domain/Creators/Sphere.hpp

```diff
@@ -125,10 +128,40 @@ namespace domain::creators {
  * \image html WedgeOrientations.png "The orientation of each wedge in a cubed
  * sphere."
  *
- * \par Inner cube sphericity
+ * #### Inner cube sphericity
  * The inner cube is a BulgedCube except if the inner cube sphericity is
  * exactly 0. Then an Equiangular or Affine map is used (depending on if it's
  * equiangular or not) to avoid a root find in the BulgedCube map.
+ *
+ * #### Time dependent maps
+ * There are two ways to add time dependent maps to the Sphere domain
+ * creator. In the input file, these are specified under the
+ * `TimeDependentMaps:` block.
+ *
+ * ##### TimeDependence
+ * You can use a simple TimeDependence (e.g.
+ * `domain::creators::time_dependence::UniformTranslation` or
+ * `domain::creators::time_dependence::RotationAboutZAxis`) to add time
+ * dependent maps. This method will add the same maps to all blocks in the
+ * domain. This method can be used with an inner cube or with an excision
+ * surface.
+ *
+ * ##### Hard-coded time dependent maps
+ * The Sphere domain creator also has the option to use some hard coded time
+ * dependent maps that may be useful in certain scenarios. This method adds the
+ * maps in `domain::creators::sphere::TimeDependentMapOptions` to the domain.
+ * Currently, the first (inner-most) shell has maps between `Frame::Grid`,
```

Comment:
Add a comment somewhere in `TimeDependentMapOptions` reminding people to update the documentation here when they change that class.